packages/neon_lints/lib/flutter.yaml

```diff
@@ -1,9 +1,228 @@
-include: package:neon_lints/dart.yaml
```

Comment:
I liked how this was extending the other one. I would like to keep it this way instead of duplicating almost everything.